roles/ncn-common-setup/files/srv/cray/resources/metal/dracut.conf.d/00-csm.conf

```diff
@@ -0,0 +1,37 @@
+# Add modules to ensure our RAID devices and netboots work, omit modules that are not used or not installed.
+add_dracutmodules+="mdraid"
+force_drivers+="raid1"
+
+# These are not found and not necessary, omit them to change their "not found" errors into messages for intentionally omitted.
+omit_dracutmodules+="btrfs cifs dmraid dmsquash-live-ntfs fcoe fcoe-uefi iscsi modsign multipath nbd nfs ntfs-3g" 
+omit_drivers+="ecb hmac md5"
+
+# kdump.service will automatically use "--compress=xz -0 --check=crc32", but our normal builds should use xz to match.
+compress="xz"
+
+# kdump.service will build with `--hostonly` regardless, but we also want local installs of the 
+# kernel and post-triggers from RPMs to generate small initrds. Our contexts that create initrds for
+# PXE and DISK booting will need to pass `--no-hostonly` and `--no-hostonly-cmdline`.
+hostonly="yes"
```

Comment:
Debating if I want to set this to `no`.. but as long as we use `--no-hostonly` and `--no-hostonly-cmdline` in our `create-kis-artifacts.sh` and `create-ims-initrd.sh` we can leave this set to `yes`.